web-ui/src/pages/PermissionsPage.jsx

```diff
@@ -66,14 +66,28 @@ const EditPermissionsPage = () => {
   const { csrf } = state;
   const roles = selectRoles(state);
   const hasPermission = selectHasPermissionAssignmentPermission(state);
-  const [selectedRole, setSelectedRole] = useState(
-    roles && roles.find(() => true)
-  );
+  const [selectedRole, setSelectedRole] = useState(roles[0]);
   const [categoriesList, setCategoriesList] = useState([]);
   const [rolePermissionsList, setRolePermissionsList] = useState([]);
   const [rolePermissions, setRolePermissions] = useState([]);
   const [refresh, setRefresh] = useState(true);
 
+  useEffect(() => {
```

Comment:
On page load this gets the selected role from a query parameter. If not found, it defaults to the first available role.